### DISCUSSION TITLE
Caddy running only on IPv6 tcp6

### DISCUSSION THREAD

**Discussion by pitbullcan** [postalserver/postal#1543] [Help with using Postal]
I have issue of web server only running on ipv6 even when I do not have IPv6 on my VPS.
`root@postal:~# netstat -tulpn | grep :80                                        tcp6       0      0 :::80                   :::*                    LISTEN      6447/caddy
`
Please any help on how to fix.

**Comment by willpower232** [comment]
Its not immediately obvious but anything listening on IPv6 ports is automatically listening on the same IPv4 ports so you shouldn't have any problems accessing the web UI presuming any firewall in place allows it.

**Reply by pitbullcan** [reply]
Thanks for your response, I have been able to fix it. I edited postal.yml and changed the web server bind address to 0.0.0.0
and it worked.
